src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -72,6 +72,11 @@
             }
         }
 
+        .bcs-activity-feed-annotation-activity.bcs-is-focused {
+            padding-left: 21px;
```

Comment:
Do we need to override the animation? I think this results in the focused annotation activity background-color fading back to transparent

Reply:
I assumed that we would want to keep the focus behaviors similar to the other cards on the activity feed. I can follow up with design.

Reply:
Spoke with design and we should keep the background blue, can I follow up with another PR as this one is already getting pretty big?